lms/templates/api_admin/catalog-results.underscore

```diff
@@ -0,0 +1,6 @@
+<h2 class="api-subheading"><%- gettext("This catalog's courses:") %></h2>
+<ul class="preview-results-list">
```

Comment:
per our offline conversation, I think this is fine based on the use case for preview.  We can update if more use cases come up